osquery/tables/system/darwin/nfs_shares.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <vector>
+#include <string>
+
+#include <boost/algorithm/string/join.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+
+#include <glog/logging.h>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+
+namespace osquery {
+	namespace tables {
+
+		QueryData parseNfsSharesContent(const std::string& content) {
+			QueryData results;
+
+			for (const auto& i : split(content, "\n")) {
+				auto line = split(i);
+				if (line.size() == 0 || boost::starts_with(line[0], "#")) {
+					continue;
+				}
+				std::vector<std::string> lineExports;
+				unsigned int readonly = 0;
+				int indexOfOptions = -1;
+
+				for (std::vector<std::string>::iterator iter = line.begin();
+					 iter != line.end();
+					 ++iter){
+					indexOfOptions++;
+					if((*iter)[0] == '/'){
+						lineExports.push_back(*iter);
+						
+					}else{
+						break;
+					}
+				}
+				for(std::vector<std::string>::iterator iter = line.begin()+indexOfOptions;
+					 iter != line.end();
+					 ++iter){
+					if(iter->compare("-ro") == 0|| iter->compare("-o") == 0){
```

Comment:
add a space in between `0` and `||`, although clang-format should take care of that